wiki/Star/fr.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I've just moved it to `/wiki/Disambiguation/Star` in #5457, please do so accordingly

Reply:
same goes for `Ouendan`: #5451